bootloader/firmware/updater.c

```diff
@@ -91,3 +90,35 @@ bool fw_write_to_flash(uint32_t address, uint64_t data) {
                              address,
                              data) == HAL_OK;
 }
+
+
+#define SYSMEM_START APP_FLASH_ADDRESS
+#define SYSMEM_BOOT (SYSMEM_START + 4)
+
+const uint32_t *const sysmem_boot_loc = (uint32_t*)SYSMEM_BOOT;
+
+void fw_update_start_application() {
```

Comment:
This is pretty much a copy paste of `app_update_start` in which we jump to the bootloader. 

I need to check if this can be parameterized.